File: Latihan_b/Python/Processor.py
class Processor:

    # atribut
    __name = ""
    __price = 0

    
    # konstruktor
    def __init__(self, name, price):
        self.__name = name
        self.__price = price


    # setter
    def setName(self, name):
        self.__name = name

    def setPrice(self, price):
        self.__price = price

    # getter
    def getName(self):
        return self.__name

    def getPrice(self):
        return self.__price

File: Latihan_b/Java/Processor.java
class Processor {

    // atribut
    private String name;
    private int price;

    
    // konstruktor
    public Processor(){}

    public Processor(String name, int price)
    {
        this.name = name;
        this.price = price;
    }

    // setter
    public void setName(String name)
    {
        this.name = name;
    }

    public void setPrice(int price)
    {
        this.price = price;
    }

    // getter
    public String getName()
    {
        return this.name;
    }

    public int getPrice()
    {
        return this.price;
    }
}